src/pages/Admin/VenueEventDetails.tsx

```diff
@@ -27,6 +27,7 @@ const VenueEventDetails = ({
   const startTime = formatTimeLocalised(eventStartTime(venueEvent));
   const endTime = formatTimeLocalised(eventEndTime(venueEvent));
   const startDay = format(eventStartTime(venueEvent), "EEEE LLLL do");
+  const isEditable = !className;
```

Comment:
This logic seems hacky

Reply:
This is what we had initially when the edit/delete was in place 
Gonna check it

Reply:
fixed f68b41e